metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-controller-manager.json

```diff
@@ -329,7 +329,7 @@
             "default_timefield": "@timestamp",
             "drop_last_bucket": 1,
             "id": "61ca57f0-469d-11e7-af02-69e470af7417",
-            "index_pattern": "",
+            "index_pattern": "metricbeat-*",
```

Comment:
There is also a `default_index_pattern` param in `visState.params`, is it not used? Could we use it in the dashboard exporter to fill this `index_pattern` when empty?

Reply:
`default_index_pattern` doesn't seem to be used by Kibana hmmm 
If we are trying to use a dashboard in Metricbeat with `index_pattern` equals to `""` and the default index pattern is `filebeat-*`, then `filebeat-*` will be used as default index pattern. It did not check for `default_index_pattern` field. 

Yes I think it would be good to change dashboard exporter to fill `index_pattern` 👍 I will create a separate PR for that. Thank you!